src/go/rpk/pkg/cli/cmd/cluster/metadata.go

```diff
@@ -162,99 +162,165 @@ func printBrokers(controllerID int32, brokers kadm.BrokerDetails) {
 	}
 }
 
-func PrintTopics(topics kadm.TopicDetails, internal, detailed bool) {
-	if !detailed {
+// Used for structured output. Note: structured output is always detailed and doens't require the -d flag.
+type TopicPartitionDetail struct {
```

Comment:
These lines are particularly long, but I don't think we need to document these fields. Most of these output structs are not documented.

Reply:
Yup, removed.